en/README.md

```diff
@@ -0,0 +1,1138 @@
+# Arm Plugin Interface 2.0 Specification(2015/04/16)
+
+## Overview
+
+Hatohol Arm Plugin Interface (HAPI) 2.0 is protocol for information exchange between Hatohol server and Monitoring server plugin.
+It is constructed as JSON-RPC appllication on the communication path that is consolidated between them.
+
+The following figure express the above overview.
+
+![overview](../hapi_overview.png)
+
+## Words
+
+|words|description|
+|:---|:---|
+|SRV|Hatohol server|
+|HAP|Hatohol monitoring server plugin|
+
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" conform to [RFC2119](http://www.ietf.org/rfc/rfc2119.txt).
+
+## Involved protocol
+
+### Communication path 
+
+Use Advanced Message Queuing Protocol (AMQP) 0.9.1 as communication path.
+- AMQP 0.9.1: https://www.rabbitmq.com/resources/specs/amqp0-9-1.pdf
+
+### JSON-RPC
+
+Use JSON-RPC 2.0 as basic protocol of information exchange.
+- [JSON-RPC 2.0 (2013-01-04)](http://www.jsonrpc.org/specification)
+- [JSON(RFC4627)](https://www.ietf.org/rfc/rfc4627.txt)
+
+#### Attention
+- Encoding is UTF-8(MUST). Normalization form C(SHOULD). String can escape(MAY).
+- Enough randomness is needed from ID object that is used in request and response(SHOULD).
```

Comment:
"REQUIRED" keywords is not exists in Japanese document too.
If we add "REQUIRED", please send PR that is to add "REQUIRED" keywords to master.